tests/config_test.go

```diff
@@ -95,7 +96,7 @@ var _ = Describe("[rfe_id:899][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 					"option2": "value2",
 					"option3": "value3",
 				}
-				tests.CreateConfigMap(configMapName, testsuite.GetTestNamespace(nil), data)
+				libconfigmap.New(configMapName, testsuite.GetTestNamespace(nil), data)
```

Comment:
`libconfigmap.New` does not create the configmap, here you have to get the returned value and perform the creation.
I also suggest you to merge this and the following commit into one, so that it is clear, in a single commit, that you are dropping the utility function in favor of the new one. Thanks